packages/core/admin/ee/admin/content-manager/pages/EditView/InformationBox/InformationBoxEE.js

```diff
@@ -25,9 +26,7 @@ export function InformationBoxEE() {
   const { formatMessage } = useIntl();
   const { formatAPIError } = useAPIErrorHandler();
 
-  const {
-    workflows: { data: workflow },
-  } = useReviewWorkflows(activeWorkflowStage?.id);
+  const workflow = get('workflows.data[0]', useReviewWorkflows());
```

Comment:
That would be fine!

Reply:
There is an issue with that implementation, it will end up with `Cannot read properties of undefined (reading 'data')`.
That's partly why I have used Lodash. But since you say that FE doesn't want Lodash anymore, I will add some code. 👍

Reply:
I would personally refactor the useReviewWorkflows hook to not return the raw object from the endpoint, but to return:
{ workflows: [workflow_ids..] }

Reply:
mb I thought it was the api response, makes sense!